src/services/fetchers/MetalsDevApiFetcher.ts

```diff
@@ -47,6 +54,18 @@ export default class MetalsDevApiPriceFetcher {
           `[MetalsDevApiFetcherFetcher] resolved price per gram: ${input.metal}/${input.currency}: ${pricePerGram}`,
         );
 
+        const payload: PriceDataPayload = {
+          fetcher: FetcherName.METALS_DEV_API,
+          value: pricePerGram.toString(),
+          valueType: 'string',
+          timestamp: this.timeService.apply(),
+          feedBase: input.currency,
+          feedQuote: input.metal,
```

Comment:
base-quote is not based on API parameters, this is based on feed symbol, please make sure you using symbol in other places as well.

Reply:
Within this fetcher I cannot find the symbol, in fact this is the payload that is coming in:

```typescript
export interface MetalsDevApiInputParams {
  metal: string;
  currency: string;
}
```

Reply:
```
ARB-USD:  <<< this is symbol
  discrepancy: 1.0
  precision: 6
  inputs:
    - fetcher:
 ```

Reply:
That is correct, how can I get that value inside this method?

The previous interface is received as argument:

```typescript
 async apply(input: MetalsDevApiInputParams): Promise<number> {
```

Reply:
this was one of discussed issues on task, that we dont have it atm, you need to pass this as additional parameter to fetchers. if you want, this can be separate PR.

Reply:
Ok, that sounds reasonable since the PR is taking quite a long time right now. What do you think?

Reply:
let's do separate PR, not a problem, let's marke all this places as `TODO`